livekit-agents/livekit/agents/voice/agent_session.py

```diff
@@ -1094,6 +1094,10 @@ def _update_user_state(
         )
 
     def _user_input_transcribed(self, ev: UserInputTranscribedEvent) -> None:
+        if self.user_state == "away" and ev.is_final:
```

Comment:
similarly, should we trigger user_state changes to speaking when any input is detected?

Reply:
like firing speaking for 0.5s?

Reply:
I guess it's fine as well. though it'd be good to handle "no final transcripts" in a generic fashion.. we should time it out and just stick with interim if final never comes